.github/workflows/beam_PreCommit_GoPrism.yml

```diff
@@ -0,0 +1,89 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: PreCommit GoPrism
```

Comment:
Have you validated that this action passes on a fork of the repo?

Reply:
Technically no. But I have no reason to believe it won't work. 

This file is copied from GoPortable, with GoPortable switched with GoPrism, and the new gradle command. I've been running the new gradle command the whole time to get this working.

I also somehow ended up with a beam repo branch rather than a fork branch.... I don't know how that happened. Something clearly changed with my usual flow. That aside...

Reply:
I'll merge this in, and run the dispatch right away, and I'll be ready to disable the auto-runs if that manual kick fails.

Not ideal, but I'm confident.

Reply:
https://github.com/apache/beam/actions/runs/6226441348/job/16898968175

Reply:
Well, that failed. 

Somehow it doesn't have a go.mod file in the beam/sdks/ directory, so it can't build a prism instance.

Reply:
When. I was adding the govulncheck action it was funky about that, I had to specify the working directory to be the sdks directory and let it run there rather than specifying the path to the directory from Beam root

Reply:
Agreed. I think the issue is that it's building from the beam directory rather than beam/sdks. Changing directories before building the prism binary should sort it out.

Not entirely sure why it works locally for me, since that should be the same as well... interesting.